<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_interface</class>
 <widget class="QWidget" name="main_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
<<<<<<< HEAD
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>550</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
=======
>>>>>>> shizhonyu
  <widget class="QToolButton" name="btn_himg">
   <property name="geometry">
    <rect>
     <x>890</x>
<<<<<<< HEAD
     <y>20</y>
=======
     <y>60</y>
>>>>>>> shizhonyu
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Â§¥ÂÉè</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="function">
   <property name="geometry">
    <rect>
     <x>880</x>
<<<<<<< HEAD
     <y>120</y>
=======
     <y>200</y>
>>>>>>> shizhonyu
     <width>117</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>function</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_pi">
      <property name="text">
       <string>‰∏™‰∫∫‰ø°ÊÅØ</string>
      </property>
     </widget>
    </item>
    <item>
<<<<<<< HEAD
     <widget class="QPushButton" name="btn_login">
      <property name="text">
       <string>ÁôªÂΩï</string>
      </property>
     </widget>
    </item>
    <item>
=======
>>>>>>> shizhonyu
     <widget class="QPushButton" name="btn_set">
      <property name="text">
       <string>ËÆæÁΩÆ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clt">
      <property name="text">
       <string>ÊàëÁöÑÊî∂Ëóè</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_his">
      <property name="text">
       <string>ÂéÜÂè≤ËÆ∞ÂΩï</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="bookdiaplay">
   <property name="geometry">
    <rect>
     <x>330</x>
<<<<<<< HEAD
     <y>130</y>
=======
     <y>140</y>
>>>>>>> shizhonyu
     <width>501</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>bookdisplay</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
<<<<<<< HEAD
    <item row="0" column="4">
     <widget class="QToolButton" name="book3">
=======
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="book4">
>>>>>>> shizhonyu
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
<<<<<<< HEAD
       <iconset resource="img.qrc">
        <normaloff>:/img/book3.png</normaloff>:/img/book3.png</iconset>
=======
       <iconset>
        <normaloff>:/img/book4.png</normaloff>:/img/book4.png</iconset>
>>>>>>> shizhonyu
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
<<<<<<< HEAD
    <item row="2" column="0">
     <widget class="QToolButton" name="book4">
=======
    <item row="0" column="4">
     <widget class="QToolButton" name="book3">
>>>>>>> shizhonyu
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
<<<<<<< HEAD
       <iconset resource="img.qrc">
        <normaloff>:/img/book4.png</normaloff>:/img/book4.png</iconset>
=======
       <iconset>
        <normaloff>:/img/book3.png</normaloff>:/img/book3.png</iconset>
>>>>>>> shizhonyu
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="book1">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
<<<<<<< HEAD
       <iconset resource="img.qrc">
=======
       <iconset>
>>>>>>> shizhonyu
        <normaloff>:/img/book1.png</normaloff>:/img/book1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
<<<<<<< HEAD
    <item row="0" column="2">
     <widget class="QToolButton" name="book2">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/book2.png</normaloff>:/img/book2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
=======
>>>>>>> shizhonyu
    <item row="2" column="2">
     <widget class="QToolButton" name="book5">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
<<<<<<< HEAD
       <iconset resource="img.qrc">
=======
       <iconset>
>>>>>>> shizhonyu
        <normaloff>:/img/book5.png</normaloff>:/img/book5.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QToolButton" name="book6">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
<<<<<<< HEAD
       <iconset resource="img.qrc">
=======
       <iconset>
>>>>>>> shizhonyu
        <normaloff>:/img/book6.png</normaloff>:/img/book6.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
<<<<<<< HEAD
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
=======
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
>>>>>>> shizhonyu
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
<<<<<<< HEAD
=======
    <item row="0" column="2">
     <widget class="QToolButton" name="book2">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/img/book2.png</normaloff>:/img/book2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
>>>>>>> shizhonyu
   </layout>
  </widget>
  <widget class="QGroupBox" name="search">
   <property name="geometry">
    <rect>
     <x>330</x>
<<<<<<< HEAD
     <y>20</y>
=======
     <y>40</y>
>>>>>>> shizhonyu
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>search</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="LE_search"/>
    </item>
    <item>
     <widget class="QToolButton" name="btn_search">
      <property name="text">
       <string>üîç</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="booklistshow">
   <property name="geometry">
    <rect>
<<<<<<< HEAD
     <x>40</x>
     <y>30</y>
     <width>231</width>
     <height>551</height>
=======
     <x>0</x>
     <y>30</y>
     <width>231</width>
     <height>571</height>
>>>>>>> shizhonyu
    </rect>
   </property>
   <column>
    <property name="text">
     <string>booklist</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Â∞èËØ¥Á±ª</string>
    </property>
    <item>
     <property name="text">
      <string>ÊñáÂ≠¶Â∞èËØ¥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ë®ÄÊÉÖÂ∞èËØ¥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÁßëÂπªÂ∞èËØ¥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÊÇ¨ÁñëÂ∞èËØ¥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂéÜÂè≤Â∞èËØ¥</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>ÈùûÂ∞èËØ¥Á±ª</string>
    </property>
    <item>
     <property name="text">
      <string>‰∫∫ÊñáÁ§æÁßëÁ±ª</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÊïôËÇ≤ËÄÉËØïÁ±ª</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ëâ∫ÊúØËÆæËÆ°Á±ª</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂÅ•Â∫∑ÂÖªÁîüÁ±ª</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÊóÖÊ∏∏Âú∞ÁêÜÁ±ª</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>ÂÑøÁ´•ËØªÁâ©Á±ª</string>
    </property>
    <item>
     <property name="text">
      <string>ÁªòÊú¨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Á´•ËØù</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Âä®Êº´</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÁßëÊôÆ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÊàêÈïøÊïôËÇ≤</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>ÁßëÂ≠¶ÁßëÊäÄÁ±ª</string>
    </property>
    <item>
     <property name="text">
      <string>Ëá™ÁÑ∂ÁßëÂ≠¶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Â∑•Á®ãÁßëÊäÄ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂåªÂ≠¶ÂÅ•Â∫∑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ËÆ°ÁÆóÊú∫ÁΩëË∑Ø</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>ÁªèÊµéÁÆ°ÁêÜÁ±ª</string>
    </property>
    <item>
     <property name="text">
      <string>Ëê•ÈîÄÁÆ°ÁêÜ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ë¥¢Âä°‰ºöËÆ°</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÊäïËµÑÈáëËûç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‰ºÅ‰∏öÁÆ°ÁêÜ</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>ÊñáÂåñËâ∫ÊúØÁ±ª</string>
    </property>
    <item>
     <property name="text">
      <string>ÂéÜÂè≤ÊñáÂåñ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Èü≥‰πêËàûËπà</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÁæéÊúØ‰π¶Ê≥ï</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÁîµÂΩ±ÊàèÂâß</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>ÂÜõ‰∫ãÁ±ª</string>
    </property>
    <item>
     <property name="text">
      <string>ÂÜõ‰∫ãÂéÜÂè≤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂÜõ‰∫ãÁêÜËÆ∫</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂÜõ‰∫ãÁßëÊäÄ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂÜõ‰∫ãÂ∞èËØ¥</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Âì≤Â≠¶ÂÆóÊïôÁ±ª</string>
    </property>
    <item>
     <property name="text">
      <string>Âì≤Â≠¶ÊÄùÊÉ≥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂÆóÊïôÊñáÂåñ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂøÉÁÅµÁîüÊ¥ª</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Êñ∞Èóª‰º†Êí≠Á±ª</string>
    </property>
    <item>
     <property name="text">
      <string>Êñ∞ÈóªÊó∂‰∫ã</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‰º†Â™íÁ†îÁ©∂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂπøÂëäËê•ÈîÄ</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Ê≥ïÂæãÊîøÊ≤ªÁ±ª</string>
    </property>
    <item>
     <property name="text">
      <string>ÊîøÊ≤ªÁêÜËÆ∫</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ê≥ïÂæãÂÆûÂä°</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ÂÖ¨ÂÖ±ÁÆ°ÁêÜ</string>
     </property>
    </item>
   </item>
  </widget>
<<<<<<< HEAD
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
=======
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1001</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Header"/>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1001</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="toolLayout"/>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>‰∏ä‰∏ÄÈ°µ</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>560</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>‰∏ã‰∏ÄÈ°µ</string>
   </property>
  </widget>
 </widget>
 <resources/>
>>>>>>> shizhonyu
 <connections/>
</ui>
